package.json

```diff
@@ -24,9 +24,18 @@
     "npm-watch": "^0.6.0",
     "pg": "^7.14.0",
     "pg-hstore": "^2.3.3",
+    "react-select": "^3.0.8",
     "sequelize": "^5.21.2",
     "swagger-tools": "^0.10.4",
     "swagger-ui-express": "^4.1.2",
     "yamljs": "^0.3.0"
+  },
+  "devDependencies": {
+    "eslint": "^6.7.1",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.16.0",
+    "prettier": "^1.19.1",
+    "prettier-eslint": "^9.0.1"
```

Comment:
I think this is added in the wrong package.json file, it should only be in the file belonging to the web-page, not the whole TheSpoon project. After pulling your code, webstorm suggests to lint the mobileApp as well, but that is wrong since we do not use React :)